tests/Models/User.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace LangleyFoxall\LaravelAuthenticationLog\Tests\Models;
+
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Database\Eloquent\Model;
+
+class User implements Authenticatable
+{
+    public $id;
+
+    /**
+     * Get the name of the unique identifier for the user.
+     *
+     * @return string
+     */
+    public function getAuthIdentifierName(){}
```

Comment:
```suggestion
    public function getAuthIdentifierName(){
      // no-op -- This is used only for testing to satisfy the specification of Authenticatable
    }
```

Reply:
Apply that to each function -- You may not need to doc comment above the function once this is done